### DISCUSSION TITLE
Termostati virtuali per aree di casa

### DISCUSSION THREAD

**Discussion by esposimo** [indomus/forum#2739] [HUB: Home Assistant]
Ciao a tutti ragazzi, scrivo per avere informazioni circa una idea di "termostati virtuali" per aree di casa.
Mi spiego: solitamente in una casa dove c'è un termostato e la gestione comandata della caldaia, funziona pressapoco così (correggetemi pure)

Termostato al centro della casa che regola l'accensione/spegnimento della caldaia in base alla temperatura rilevata (temperatura, ripeto, misurata al centro della casa)
Tralasciando il fatto che alcune caldaie possono anche essere regolate dal termostato e quindi mantenere la fiamma bassa e i termosifoni costanti, la mia idea si basa su una configurazione classica non domotica (da renderla con HA)

Ovvero caldaia + termosifoni classici.

Applicando un relè smart a contatto pulito alla caldaia, il primo step è fatto, ovvero posso accendere/spegnere la caldaia (per la parte termosifoni) ma non regolare la sua temperatura (mettiamo che sia fissata su 30 gradi)

Per ogni termosifone, metto una valvola termostatica smart e un rilevatore di temperatura (o più sensori se la stanza e grande e ne creo uno virtuale su HA che faccia la media). Dato che le stanze si riscaldano in tempi diversi, l'idea è di

- Chiudere la valvola smart (magari portandola al 20%) nel momento in cui l'area arriva a temperatura desiderata.
- Fare questo per ogni area della casa
- Chiudere la caldaia quando tutte le aree della casa hanno raggiunto la temperatura desiderata.

Potrebbe essere anche utile nei casi in cui si è fuori casa, bisogna rientrare solo per una doccia rapida e si desidera riscaldare solo il bagno

E' una configurazione sbagliata secondo voi? E' gestibile ?

**Comment by 50494554524F** [comment]
anche io faccio così... il maggior risultato lo ottieni dove le porte son chiuse, altrimenti l'uso delle valvole è quasi irrilevante

**Reply by esposimo** [reply]
Si avevo pensato anche alla questione porte e finestre chiuse...da gestire con dei sensori per avvisarti, in caso di accensione, di andarle a chiudere 😅

**Comment by snakuzzo** [comment]
Non so che tipo di teste termostatiche hai intenzione di usare, ma solitamente le teste smart non possono essere regolate in percentuale dall'utente. L'utente imposta una temperatura, puoi è la logica della stessa testa che porta ad un apertura totale o parziale

Quindi...ok per l'utilizzo di sensori per recuperare una temperatura più realistica in punti ben precisi, ma devi considerare che le teste smart ragionano per conto loro, con la temperatura rilevata da loro...e non da HA.
Tralasciando il discorso apertura percentuale, che non credo proprio sia fattibile, per fare quello che dici tu hai due soluzioni:
- utilizzare delle teste termostatiche che possano essere impostate su OFF o su ON (senza temperatura target, una sorta di BOOST)
- utilizzare delle teste termostatiche su cui possa essere impostato un offset (da valutare se regolarlo dinamicamente con automazione lato HA basata sulla temperatura rilevata dal sensore del climate virtuale oppure manualmente sperando che con una semplice regolazione una tantum dell'offset possa bastare...ma ne dubito)

Nel caso non fosse possibile fare ciò, ti faccio un esempio di quello che potrebbe succedere::

- Imposti una temperatura di 23° sul climate virtuale
- tramite automazione scatta il trigger per portare la testa tesrmostatica a 23° e la valvola si apre
- la testa termostatica raggiunge i 23° mentre il sensore del climate virtuale rileva ancora 20°

Risultato:

- La testa chiude la valvola e non riscalda più...il tutto mentre su HA tu hai ancora il climate acceso, che probabilmente non raggiungerà mai la temperatura target di 23°...tutto sto giro diventerebbe totalmente inutile.

**Reply by Scialla** [reply]
Aggiungo per completezza che, se hai una caldaia SENZA la funzione di ricircolo, almeno un termosifone lo devi lasciare SENZA valvola, ovvero deve rimanere SEMPRE aperto altrimenti, nel caso tutte le valvole dei termosifoni siano chiuse e la caldaia sia ancora accesa, questa ti va in blocco e potrebbe pure fare danni....

Te lo dico perché ci sono passato e ho dovuto togliere una valvola smart da un termosifone perché a volte, anche se il termostato dice alla caldaia di spegnersi, c'è un ritardo strutturale nel quale tutte le teste chiudono e la caldaia è ancora accesa per una decina di secondi o più.... a me la caldaia faceva dei rumoracci impressionanti per poi andare in blocco (fortunatamente senza danni)

**Reply by esposimo** [reply]
Rispondo ad entrambi

@Scialla già so di questa cosa e l'avevo già preventivata. Ad oggi la caldaia devo ancora montarla, ad ogni modo è una Ariston da 24kw , questa è la scheda tecnica (https://www.schede-tecniche.it/schede-tecniche-caldaie/ARISTON-manuale-tecnico-caldaia-a-condensazione-HS-PREMIUM-24-30-EU.pdf). Non ho letto tutto il manuale, quindi non so se è provvista di tale funzione. Ipotizzo di si (anzi spero) perchè ha anche la parte wifi (acquistabile separatamente)


@snakuzzo userei le valvole shelly, che nella documentazione riportano sia la possibilità di inviare comandi su mqtt con indicazione della temperatura, che della percentuale (https://shelly-api-docs.shelly.cloud/gen1/#shelly-trv-overview). Tra l'altro è indicato che se imposti una percentuale arbitraria viene meno la considerazione della temperatura impostata. Infine, queste valvole possono attingere il valore della temperatura ambientale anche da sensori esterni, e non dal loro sensore interno (cosa utile visto che il sensore probabilmente è la prima cosa a scaldarsi)

**Reply by snakuzzo** [reply]
Ah ok...comunque se riesci a farli basare su un sensore esterno, credo sia comunque inutile gestire l'apertura percentuale manualmente. Come detto prima, hanno già una logica loro... perché andare a mettere mano su quello?

**Reply by esposimo** [reply]
@snakuzzo perché la mia idea è di tenerli al 10/20% per un po' quando la stanza arriva a temperatura, per poi chiudere. O regolarli per dare priorità ad una stanza piuttosto che un'altra. Anche per giocarci un po' ecco
